packages/react/spec/auto/PolarisAutoTable.stories.jsx

```diff
@@ -14,7 +15,7 @@ export default {
       // 👇 Make it configurable by reading the theme value from parameters
       return (
         <Provider api={api}>
-          <AppProvider>
+          <AppProvider i18n={translations}>
```

Comment:
This is required to get the text into the cancel button. Unfortunately, Shopify does not have a default i18n language which leads to empty strings when one is not explicitly given